app/views/pages/robots.text.erb

```diff
@@ -1,4 +1,4 @@
-<% if TradeTariffFrontend.production? %>
+<% if Rails.env.production? && request.domain == "www.trade-tariff.service.gov.uk" %>
```

Comment:
if it possible to encapsulate this in a method on the TradeTariffFrontend module?

I see we already have a production? method, perhaps a production_domain? method?